qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/ClientCancelPendingCommandTest.java

```diff
@@ -9,21 +9,26 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import io.camunda.zeebe.client.ZeebeClient;
 import io.camunda.zeebe.gateway.metrics.LongPollingMetrics;
 import io.camunda.zeebe.it.clustering.ClusteringRuleExtension;
+import io.camunda.zeebe.qa.util.JobStreamServiceAssert;
 import java.time.Duration;
+import java.util.UUID;
 import org.awaitility.Awaitility;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 final class ClientCancelPendingCommandTest {
   @RegisterExtension
-  final ClusteringRuleExtension cluster = new ClusteringRuleExtension(1, 1, 1, cfg -> {});
+  private static final ClusteringRuleExtension CLUSTER =
+      new ClusteringRuleExtension(1, 1, 1, cfg -> {});
+
+  private final ZeebeClient client = CLUSTER.getClient();
 
   @Test
   void shouldCancelCommandOnFutureCancellation() {
```

Comment:
:question: Do we still need to keep this test?

Reply:
Good question. It was already a bit hacky to use the metric for assertions. I think we should because eventually we should handle cancellation propagation for all commands.